user_claims_test.go

```diff
@@ -227,6 +227,42 @@ func TestUserValidation(t *testing.T) {
 	if vr.IsEmpty() || len(vr.Issues) != 1 || !vr.IsBlocking(true) {
 		t.Error("bad limit should be invalid")
 	}
+
+	uc.Limits.Times = []TimeRange{}
+	uc.Permissions.Pub.Allow.Add("bad subject")
+	vr = CreateValidationResults()
+	uc.Validate(vr)
+
+	if vr.IsEmpty() || len(vr.Issues) != 1 || !vr.IsBlocking(true) {
+		t.Error("bad permission should be invalid")
```

Comment:
t.Error() will mark the test as failed, but will continue execution of the test. Does that make sense? I personally never use t.Error() because any error condition in my opinion invalidates the test.
(I won't repeat comment on all t.Error() occurrences).

Reply:
yes it is required - the test is mucking with the model further down to catch other kinds of errors - likely it should many different tests but it isn't.